generic_lockfile.yaml

```diff
@@ -0,0 +1,10 @@
+metadata:
+    version: "1.0"
+artifacts:
+    - download_url: https://github.com/cachito-testing/cachi2-generic/archive/refs/tags/v1.0.0.zip
+      checksums:
+        sha256: 4fbcaa2a8d17c1f8042578627c122361ab18b7973311e7e9c598696732902f87
+    - download_url: https://github.com/cachito-testing/cachi2-generic/archive/refs/tags/v2.0.0.zip
+      target: archive.zip
+      checksums:
+        sha256: 386428a82f37345fa24b74068e0e79f4c1f2ff38d4f5c106ea14de4a2926e584
```

Comment:
Here you'd put in the files you need, these are just examples.
- download_url is the url the files is available from
- target is optional and specifies a subpath the file will be saved as. If not defined, it will be derived from the url
- checksums is a list of pairs of algorithm and hash, that is necessary to verify file identity

Reply:
Those would be the files that you download from the internet during the build, which would violate the hermetic build condition. Having a brief look into your [Dockerfile, I would say it's these files](https://github.com/red-hat-data-services/trustyai-explainability/blob/main/Dockerfile.konflux#L32)

Reply:
should it be this one: https://github.com/red-hat-data-services/RHOAI-Build-Config/blob/rhoai-2.16/trustyai-pig-builds/build-config.yaml or https://gitlab.cee.redhat.com/project-ncl/utils.git as we have updated tekton file for PNC https://github.com/red-hat-data-services/trustyai-explainability/blob/rhoai-2.16/.tekton/odh-trustyai-service-v2-16-push.yaml#L284C4-L392C1